src/components/footer/footer.css

```diff
@@ -0,0 +1,142 @@
+.footer .links {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  text-align: center;
+  position: relative;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.39);
+  color: white;
+  height: auto;
+  width: 100vw;
+  padding-top: 2vh;
+  padding-bottom: 2vh;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+
+.wrapper {
+  display: flex;
+  justify-content: center;
+}
+
+.wrapper .icon {
+  position: relative;
+  background-color: #fff;
+  border-radius: 50%;
+  padding: 15px;
+  margin: 10px;
+  width: 40px;
+  height: 40px;
+  font-size: 18px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  box-shadow: 0 10px 10px rgba(0, 0, 0, 0.1);
+  cursor: pointer;
+  transition: all 0.2s cubic-bezier(0.68, -0.55, 0.265, 1.55);
+}
+
+.wrapper .toolTip {
+  display: visible;
```

Comment:
Does visible really exist ?

Reply:
Thanks for this comment. It's a mistake. I fix it.